cmd/checkpoint/main.go

```diff
@@ -42,6 +43,8 @@ var (
 )
 
 func main() {
+	flag.Set("logtostderr", "true")
```

Comment:
@pbx0 This is why you did not find any checkpoint logging.

I spent quite a while to figure out why a few days ago... glog is crazy.